snap/README.md

```diff
@@ -227,30 +226,19 @@ not addressable from another system). In order to make a service accessible remo
 
 ##### JWT tokens
 
-Before the API Gateway can be used, a user and group must be created, and a JWT access token generated. This can be accomplised via the
-`secrets-config` command, or by using `snap set` commands.
+Before the API Gateway can be used, a user and group must be created and a JWT access token generated.
 
-The first step is to add a user. You need to create a public/private keypair, which can be done with
+1. The first step is to create a public/private keypair for the new user, which can be done with
 
 ```bash
 # Create private key:
-$ openssl ecparam -genkey -name prime256v1 -noout -out private.pem
+openssl ecparam -genkey -name prime256v1 -noout -out private.pem
 
 # Create public key:
-$ openssl ec -in private.pem -pubout -out public.pem
+openssl ec -in private.pem -pubout -out public.pem
```

Comment:
openssl ec -in private.pem -pubout -out public.pem
Can't open private.pem for reading, Permission denied
140544944436544:error:0200100D:system library:fopen:Permission denied:../crypto/bio/bss_file.c:69:fopen('private.pem','r')
140544944436544:error:2006D002:BIO routines:BIO_new_file:system lib:../crypto/bio/bss_file.c:78:

(I think it need to be used with `sudo`)

Reply:
As above... you should run this command somewhere in a local directory that you have write access to. You shouldn't need to use sudo.